<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="topLeftGraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="topLeftAirTemp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>490</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Air temp: 0 C</string>
    </property>
   </widget>
   <widget class="QLabel" name="topLeftRoadTemp">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Road temp: 0 C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lowLeftRoadTemp">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Road temp: 0 C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lowLeftAirTemp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>490</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Air temp: 0 C</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="lowLeftGraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="topRightRoadTemp">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>0</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Road temp: 0 C</string>
    </property>
   </widget>
   <widget class="QLabel" name="topRightAirTemp">
    <property name="geometry">
     <rect>
      <x>642</x>
      <y>0</y>
      <width>488</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Air temp: 0 C</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="topRightGraphicsView">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="lowRightGraphicsView">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lowRightAirTemp">
    <property name="geometry">
     <rect>
      <x>642</x>
      <y>360</y>
      <width>488</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Air temp: 0 C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lowRightRoadTemp">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>360</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Road temp: 0 C</string>
    </property>
   </widget>
   <zorder>topRightGraphicsView</zorder>
   <zorder>lowLeftGraphicsView</zorder>
   <zorder>topLeftGraphicsView</zorder>
   <zorder>topLeftAirTemp</zorder>
   <zorder>topLeftRoadTemp</zorder>
   <zorder>lowRightGraphicsView</zorder>
   <zorder>lowRightAirTemp</zorder>
   <zorder>lowRightRoadTemp</zorder>
   <zorder>lowLeftRoadTemp</zorder>
   <zorder>lowLeftAirTemp</zorder>
   <zorder>topRightRoadTemp</zorder>
   <zorder>topRightAirTemp</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
